client/src/document/index.scss

```diff
@@ -325,10 +325,13 @@ iframe {
   &[src*="https://test262.report"],
   &.nobutton,
   &.sample-code-frame {
-    background: #fff;
-    border: 1px solid var(--border-primary);
-    border-radius: var(--elem-radius);
     width: 100%;
+    background: #fff;
+    border: none;
+    border-radius: calc(var(--elem-radius) / 2);
+    outline: 1px solid var(--border-primary);
```

Comment:
could we just do the outline and not having:
- `border-radius` potentially cutting away things without offset
- `outline-offset`potentially overlapping
- `box-shadow` no need to add

Reply:
> `border-radius` potentially cutting away things without offset

Not really. We are setting border radius to only 2px.
The `--elm-radius` is 0.25rem on the entire site. So we are setting the radius to only 0.25rem/2 = 0.125rem = 2px. The 2px doesn't cut anything, because it almost equals to line spacing:
**Without the radius**
![without_radius](https://user-images.githubusercontent.com/87750369/184156547-4f1a0132-7858-47e8-a861-757142ac04a7.png)

**With the radius**
![with_radius](https://user-images.githubusercontent.com/87750369/184156613-dc9ad21b-1b78-410a-94f9-bdac76547fbc.png)

However, the outline does get noticeable round corner because the radius increases due to the `outline-offset: 0.25rem` : )
![outlineRadius](https://user-images.githubusercontent.com/87750369/184157994-dd96535b-b006-4321-b69b-1d371a0656f8.png)

***
> `outline-offset`potentially overlapping

The livesamples are put only in three types of places:
1. After heading: (this is `### Result`)
![after_result](https://user-images.githubusercontent.com/87750369/184163162-231001df-516e-4f7c-b4f5-4cbc9d400438.png)

2. After paragraph:
![after_paragraph](https://user-images.githubusercontent.com/87750369/184163200-41dc3707-172e-46b8-851f-560b564561fc.png)

3. Inside a table: (notice the check box row adjusted itself)
![inside_table](https://user-images.githubusercontent.com/87750369/184163232-d302a7f1-0337-4444-a20f-f58465544e48.png)

In all the cases sufficient margin is available and it doesn't overlap.

***

> `box-shadow` no need to add

The shadow is needed in the dark mode to fill the gap between the outline and the iframe edge: 
![outlineRadius](https://user-images.githubusercontent.com/87750369/184157994-dd96535b-b006-4321-b69b-1d371a0656f8.png)